docker-compose.prod.yml

```diff
@@ -0,0 +1,54 @@
+version: "2"
+services:
+
+  db:
+    extends:
+      file: ./docker/common.yml
+      service: db
+    volumes:
+      - ./data/pgdata:/var/lib/postgresql/data/pgdata
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+
+  redis:
+    image: redis:3.2.1
+    volumes:
+      - ./data/redis:/data
+
+  web:
+    build: .
+    command: uwsgi /code/listenbrainz-server/docker/uwsgi.ini
+    volumes:
+      - .:/code/listenbrainz-server
+      - ./data/app:/data
+    ports:
+      - "3031:3031"
+    links:
+      - redis
+      - db
+    depends_on:
+      - redis
+      - db
+
+  consumer:
+    build: 
+      dockerfile: redis-consumer/Dockerfile
+      context: .
+    command: python redis-consumer/redis-consumer.py
+    volumes:
+      - .:/code/listenbrainz-server
+    links:
```

Comment:
Why do you have a links section in the prod version, but not in the normal version?